Dockerfile.release

```diff
@@ -30,21 +32,25 @@ FROM ${RELEASE_DOCKER_BASE_IMAGE} AS release
 
 ARG USER=erigon \
     GROUP=erigon \
+    UID_ERIGON \
+    GID_ERIGON \
     APPLICATION \
     EXPOSED_PORTS
 
+STOPSIGNAL 2
```

Comment:
What is this for?

Reply:
We want that erigon is shut down `gracefully` when its docker container receives the stop command.
This corresponds to SIGINT (i.e. `kill -2`) when erigon runs as a normal host process.

Reply:
Thanks, @michelemodolo for answering 👍 

We expect erigon will leave good file/db state after stopping by signal 2.
Other signals might leave problems for next run.